applications/plugins/SofaMatrix/src/SofaMatrix/FillReducingOrdering.inl

```diff
@@ -23,13 +23,16 @@
 
 #include <SofaMatrix/FillReducingOrdering.h>
 
+#include <Eigen/Sparse>
+
 namespace sofa::component::linearsolver
 {
 
 template <class DataTypes>
 FillReducingOrdering<DataTypes>::FillReducingOrdering()
     : l_mstate(initLink("mstate", "Mechanical state to reorder"))
     , l_topology(initLink("topology", "Topology to reorder"))
+    , d_orderingMethod(initData(&d_orderingMethod, sofa::helper::OptionsGroup(2, "Metis", "AMD"), "orderingMethod", "Ordering method. AMD means Approximate minimum degree."))
```

Comment:
If AMD means for something... is there a possible hint for the metis one ?

Reply:
Metis is the library

Reply:
I've changed the options. Let me know what you think